ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -1120,6 +1120,14 @@ periodics:
               test:
               - ref: clusterbot-wait
               workflow: openshift-e2e-$(CLUSTER_TYPE)-sdn
+          - as: launch-dualstack
+            steps:
+              cluster_profile: vsphere
```

Comment:
This should use `$(CLUSTER_PROFILE)` instead of being hardcoded.

Reply:
If you do want to use the `vsphere` instead of `vsphere-clusterbot` profile, we can leave this hardcoded. Just wanted to make sure that having it different was intentional.

Reply:
Yeah so that was my intention, to use "any" cluster and not only `vsphere-clusterbot`; in principle anything that can match so that the `launch vsphere,dualstack` lane has as much as possible resources

Reply:
we run specific variants of clusterbot for vSphere in different lease pools.  That is the reason we don't ${CLUSTER_PROFILE} consistently.